python/tests/test_build_disk_index.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+import time
+import argparse
+import diskannpy
+from diskannpy import Metric, Parameters, DiskANNFloatIndex
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('data_path', type=str, help='Path to the input base set of vectors.')
+parser.add_argument('save_path', type=str, help='Path to the built index.')
+parser.add_argument('R', type=int, help='Graph degree.')
+parser.add_argument('L', type=int, help='Index build complexity.')
+parser.add_argument('B', type=float, help='Memory budget in GB for the final index.')
+parser.add_argument('M', type=float, help='Memory budget in GB for the index construction.')
+parser.add_argument('T', type=int, help='Number of threads for index construction.')
+
```

Comment:
We have more arguments for this tool (See workflows folder for doc). Do we plan to expose them here later? 

On a related note, would there be documentation for the python interface or is the expectation that users read this file. One possibility to add some notes to the commandline interface documentation (workflows/SSD_index.md)

Reply:
So these aren't really even unit tests, they're more examples.  This whole tests folder needs to be reworked to give us deterministic hands-off testing.  I was going to leave it alone for now until we could really revisit it, along with adding the python quality-of-life stuff we'd want